demos/interactive_face_detection_demo/interactive_face_detection.hpp

```diff
@@ -100,8 +100,20 @@ static const char no_wait_for_keypress_message[] = "Do not wait for key press in
 /// @brief Message do not show processed video
 static const char no_show_processed_video[] = "Do not show processed video.";
 
+/// @brief Message do not show face bounding boxes
+static const char no_show_face_bbox_message[] = "Do not show face bounding boxes on processed video.";
+
+/// @brief Message do not show performance numbers on processed video
+static const char no_show_perf_message[] = "Do not show performance numbers on processed video.";
+
+/// @brief Message enable face blurring
+static const char blur_faces_message[] = "Enable face blurring.";
```

Comment:
This flag is not documented in readme. And since it's a dedicated flag, I'd make it `int`, not `bool`. The value can be the sigma of the filter.

Reply:
fixed